README.md

```diff
@@ -1,55 +1,59 @@
 # Private Box
-Alternative client for Box Critters.
-![screenshot](https://cdn.discordapp.com/attachments/632604108024578059/633333071114141696/unknown.png)
-# Table of Contents
-TODO: Optionally, include a table of contents in order to allow other people to quickly naviagte especially long or detailed READMEs.
+Alternative client and server for Box Critters.
+![screenshot](https://cdn.discordapp.com/attachments/567030108003631108/648901762748776448/private_box.png)
 
-## Getting Started
-These instructions will get you a copy of the project up and running on your local machine. See deployment for notes on how to deploy the project on a live system.
+## Getting started
 
 ### Prerequisites
-(Preperatrion to install)
+To use Private Box you need:
+* Ruby >= 2.4.0
+* Node.js >= 10.16.3
 
```

Comment:
Also, bundle? https://rubygems.org/gems/bundler/versions/1.11.2

Reply:
Bundle is installed when the user runs `make deps` or `_start_client.bat`